### DISCUSSION TITLE
hd_render and hd_usd2rdl are giving me a "Unable to load Moonray render plugin" or "Unable to load the requested renderer "Moonray" Available renderers include:

### DISCUSSION THREAD

**Discussion by xXTR0YXx** [dreamworksanimation/openmoonray#91] [Q&A]
After having moonray and moonray_gui render RDLA files properly, the next step is to convert existing USD scenes into RDLA/B files using hd_usd2rdl but I am getting those errors. Same for hd_render. Any hint where I can point those tools to moonray?

**Comment by DJtheRedstoner** [comment]
The start of the "HdMoonRay Setup" page on the documentation mentions the environment variables that need to be set: https://docs.openmoonray.org/user-reference/tools/hydra/hdmoonray-setup/. Specifically `PXR_PLUGINPATH_NAME` is what matters here. However, it seems like the documentation is wrong and specifies `$REL/plugin/pxr/usd` as the path when that doesn't exist for me and it's `$REL/plugin/usd` for me (reported [here](https://github.com/dreamworksanimation/openmoonray/discussions/47#discussioncomment-5556196)).

**Reply by xXTR0YXx** [reply]
yes, realized that the whole $REL doesn't really work properly for me, even though everything was built according to the instructions. I had to manually fill in the path for the instructions for HYDRA. In the end I was able to make it run. So I can convert USD files into RLDA/B files now and can render using hd_render.

So the manual says: $REL/bin/rdl2_json_exporter --out $REL/shader_json/ --sparse but I had to manually use /install/openmoonray/bin/rdl2_json_exporter --out /install/openmoonray/shader_json/ --sparse

Have to admit that I suck using LINUX in general. I am trying to add LINUX to my pipeline since 1995 but always switched back to Windows

**Reply by DJtheRedstoner** [reply]
The instructions say "If the MoonRay release is in `$REL`, a typical setup looks like this:" which really means "Set the `REL` environment variable to the MoonRay location, or manually replace the paths below". If you do `export REL=/install/openmoonray` first, everything should work.
